src/libs/CardUtils.ts

```diff
@@ -37,6 +37,14 @@ function isExpensifyCard(cardID: number) {
     return card.bank === CONST.EXPENSIFY_CARD.BANK;
 }
 
+/**
+ * @param cardID
+ * @returns boolean if the cardID is in the cardList from ONYX. Includes Expensify Cards.
+ */
+function isCorporateCard(cardID: number) {
```

Comment:
I might be mistaken but isn't corporate cards external company cards and excludes expensify cards?

Reply:
I'm not sure if there's a better word for both Corporate + Expensify cards. Since an Expensify card is technically a corporate card? Is the `isCardListCard` better?